src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -3624,18 +3624,39 @@
-    ("SveSimpleVecOpTest.template",       new Dictionary<string, string> { ["TestName"] = "SveReverseElement32_long",                                                                              ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "ReverseElement32",                                                     ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Int64",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Int64",                                                                                                                            ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetInt64()",                                                                                                                                                         ["ValidateIterResult"] = "result[i] != Helpers.ReverseElement32(firstOp[i])", ["GetIterResult"] = "Helpers.ReverseElement32(leftOp[i])"}),
-    ("SveSimpleVecOpTest.template",       new Dictionary<string, string> { ["TestName"] = "SveReverseElement32_ulong",                                                                             ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "ReverseElement32",                                                     ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt64", ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "UInt64",                                                                                                                           ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetUInt64()",                                                                                                                                                        ["ValidateIterResult"] = "result[i] != Helpers.ReverseElement32(firstOp[i])", ["GetIterResult"] = "Helpers.ReverseElement32(leftOp[i])"}),
+    ("SveSimpleVecOpTest.template",       new Dictionary<string, string> { ["TestName"] = "SveReverseElement8_short",                                                                              ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "ReverseElement8",                                                      ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Int16",   ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Int16",                                                                                                                            ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetInt16()",                                                                                                                                                        ["ValidateIterResult"] = "result[i] != Helpers.ReverseElement8(firstOp[i])",  ["GetIterResult"] = "Helpers.ReverseElement8(leftOp[i])"}),
```

Comment:
Can you confirm if the only difference in `ReverseElement` lines is space?

Reply:
Yup, it's just a space that misaligned the columns.